test/fuzzers/fuzz_strings.cc

```diff
@@ -0,0 +1,135 @@
+/*
+ * A fuzzer focused on C string -> Javascript String.
+*/
+
+#include <stdlib.h>
+#include "js_native_api.h"
+#include "js_native_api_v8.h"
+#include "node.h"
+#include "node_platform.h"
+#include "node_internals.h"
+#include "node_api_internals.h"
+#include "node_url.h"
+#include "env-inl.h"
+#include "util-inl.h"
+#include "v8.h"
+#include "libplatform/libplatform.h"
+#include "aliased_buffer.h"
+#include "fuzz_helper.h"
+
+using node::AliasedBufferBase;
+
+/* General set up */
+using ArrayBufferUniquePtr = std::unique_ptr<node::ArrayBufferAllocator,
+  decltype(&node::FreeArrayBufferAllocator)>;
+using TracingAgentUniquePtr = std::unique_ptr<node::tracing::Agent>;
+using NodePlatformUniquePtr = std::unique_ptr<node::NodePlatform>;
+
+static TracingAgentUniquePtr tracing_agent;
+static NodePlatformUniquePtr platform;
+static uv_loop_t current_loop;
+static napi_env addon_env;
+
+inline napi_env NewEnv(v8::Local<v8::Context> context,
+                       const std::string& module_filename,
+                       int32_t module_api_version) ;
+
+extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv) {
+  uv_os_unsetenv("NODE_OPTIONS");
+  std::vector<std::string> node_argv{ "fuzz_string_conversion" };
+  std::vector<std::string> exec_argv;
+  std::vector<std::string> errors;
+
+  node::InitializeNodeWithArgs(&node_argv, &exec_argv, &errors);
```

Comment:
Can this just use `node::InitializeOncePerProcess()` instead? Then there's no need to repeat the initialization routines below, unless there's a reason that those have to be customized (from what I can tell, that's not necessary?)